source/core/ut_suite_manager.pkb

```diff
@@ -17,6 +17,7 @@ create or replace package body ut_suite_manager is
   */
 
   gc_suitpath_error_message constant varchar2(100) := 'Suitepath exceeds 1000 CHAR on: ';
+  gc_tag_errmsg             constant number := 450;
```

Comment:
Sonar complains about `NUMBER` type without precision - change this to `INTEGER`.